### DISCUSSION TITLE
Dotspatial C# VS2022 - ToolStrip Confusion

### DISCUSSION THREAD

**Discussion by abt5555** [DotSpatial/DotSpatial#1504] [Q&A] [answered]
I am confused with dotspatial toolstrip which is installed via NuGet Package Manager in VS2022 .NET6.0.

SpatialToolStripPanel SpatialToolStrip MenuStrip SpatialHeaderControl

How should I use them in what order?

In Form1.cs [Desing], what I see in below picture

![2023-04-05 10_55_56-](https://user-images.githubusercontent.com/14254710/230024506-0fc93696-520c-4113-88c1-d8c5406bbf7c.png)

when I run the form what I see in below picture

![2023-04-05 11_15_15-](https://user-images.githubusercontent.com/14254710/230024604-4605ad4f-2d37-41a1-ab5f-69ba9574fb4f.png)

Thanks in advance

**Comment by abt5555** [comment]
I think dotspatial doesnt support .NET6.0

**Reply by jany-tenaj** [reply, marked as answer]
Check out the [getting started guide](https://github.com/DotSpatial/DotSpatial/wiki/Getting-started).

**Reply by abt5555** [reply]
Thanks for the answer. When I dont use SpatialToolStrip, it works fine. 

Now, I am trying to add GDAL extension. I tried to do like this link https://github.com/DotSpatial/DotSpatial/wiki/Plugins 

However, I cannot find the file inside Tools folder and DotSpatial.Plugins.MapWindowProjectFileCompability file inside Windows Extensions folder.

dll download link doesnt work. at least for me. 
https://appveyor.s3.us-west-004.backblazeb2.com/mogikanin-39627/dotspatial/4-0-774/p29ee3swfcrrq4am/Source/bin/Release.zip?X-Amz-Expires=300&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=004d33b301449140000000003/20230408/us-west-004/s3/aws4_request&X-Amz-Date=20230408T143202Z&X-Amz-SignedHeaders=host&X-Amz-Signature=9d76c0d4ec688e7df28842492437295fd24cb00106ab3321197476c0c23d3956

this doesnot work for me,too https://github.com/DotSpatial/DotSpatial/issues/901
